aspnetcore/signalr/hubs.md

```diff
@@ -120,6 +120,56 @@ By default, a server hub method name is the name of the .NET method. To change t
 
 :::code language="csharp" source="~/../AspNetCore.Docs.Samples/signalr/hubs/samples/6.x/SignalRHubsSample/Snippets/Hubs/ChatHub.cs" id="snippet_HubMethodName" highlight="1":::
 
+## Inject services into your hub
+
+> [!NOTE]
+> The following feature is only available in .NET 7.0+
+
+Hub constructors can accept services from DI as parameters which can be stored in properties on the class for use in a hub method.
+
+When injecting multiple services for different hub methods or as an alternative way of writing your code, hub methods can also accept services from DI.
```

Comment:
Since we favor imperative tone and dropping 2nd person where possible ...

```suggestion
When injecting multiple services for different hub methods or as an alternative way of writing code, hub methods can also accept services from DI.
```